scripts/delivery.json

```diff
@@ -63,8 +63,33 @@
             "401": { "type": "unit" },
             "404": { "type": "unit" }
           }
+        },
+        {
+          "method": "POST",
+          "path": "/:id",
+          "request body": {
+	    "type": "delivery_form",
```

Comment:
where is this `delivery_form` defined?

Reply:
then what would be the path?

Reply:
There is a parent path defined [here](https://github.com/Bhavya-verma-knoldus/api/pull/1/files#diff-92fb1c71351e25cd5813a0379d1fad78a6f3a9c38e1b980f21d006bcc2d26247R64). That will be used.